bionic/__init__.py

```diff
@@ -1,9 +1,17 @@
 from .flow import Flow, FlowBuilder  # noqa: F401
 from .decorators import (  # noqa: F401
-    version, output, outputs, docs, gather, persist, memoize, pyplot,
-    immediate, changes_per_run
+    version,
```

Comment:
Not related to your PR but this really makes me want to add `isort` to sort the imports as well